examples/platform_services/android/app/src/main/java/com/example/flutter/ExampleActivity.java

```diff
@@ -0,0 +1,113 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package com.example.flutter;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.location.LocationManager;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import io.flutter.view.FlutterMain;
+import io.flutter.view.FlutterView;
+
+import java.io.File;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class ExampleActivity extends Activity {
+    private static final String TAG = "ExampleActivity";
+
+    private FlutterView flutterView;
```

Comment:
We should have `io.flutter.FlutterActivity` expose its `flutterView`. Then this one could extend `FlutterActivity`, and we wouldn't have to deal with the basic initialization. And we wouldn't need to have a layout .xml file. That's for another change, though, since it'll require changes in the engine.